### PR TITLE
communication column has been dropped from takss table 10 years ago

### PR DESCRIPTION
**Pull request by trasher**
see https://github.com/glpi-project/glpi-inventory-plugin/commit/d52f37fa927a582bfad4a39288fd9447bb324ccb

### CODE REVIEW

**Review by cedric-anne — APPROVED** [review]

**Review by cedric-anne — CHANGES_REQUESTED** [review]
See comment.

File: inc/networkinventory.class.php [hunk 1]
```diff
@@ -650,13 +636,12 @@ public function run($jobstate)
...
     * @global object $DB
     * @param integer $nb_computers
-    * @param string $communication
     * @param string $subnet
     * @param string $ipstart
     * @param string $ipend
     * @return array
     */
-    public function getAgentsSubnet($nb_computers, $communication, $subnet = '', $ipstart = '', $ipend = '')
+    public function getAgentsSubnet($nb_computers, $subnet = '', $ipstart = '', $ipend = '')
```
[the remark this answers is not in the document]
Comment (cedric-anne) [reply inc/networkinventory.class.php:644]: I was wrong, see next comment.

File: inc/networkinventory.class.php [hunk 2]
```diff
@@ -688,77 +673,6 @@ public function getAgentsSubnet($nb_computers, $communication, $subnet = '', $ip
...
-        foreach ($iterator as $data) {
-            if ($communication == 'push') {
-                if ($pfTaskjob->isAgentAlive("1", $data['a_id'])) {
-                    if (!in_array($a_agentList, $data['a_id'])) {
-                         $a_agentList[] = $data['a_id'];
-                        if (count($a_agentList) >= $nb_agentsMax) {
-                            return $a_agentList;
-                        }
-                    }
-                }
```
[the remark this answers is not in the document]
Comment (trasher) [reply inc/networkinventory.class.php:751]: Hum... This part of code relies on `isAgentAlive()` method which has been removed 10 years ago as well... This whole method seems never called IRL.

**Review by cedric-anne — APPROVED** [review]
Seems OK.

**Pull request merged**